<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumbersCalculator</class>
 <widget class="QWidget" name="NumbersCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelElementA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Elemento A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditElementA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Formato: &quot;a, b&quot; oppure &quot;a, b, c, d&quot;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>20</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditElementB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Formato: &quot;a, b&quot; oppure &quot;a, b, c, d&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelElementB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Elemento B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditElementC">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Risultato</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelElementC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Elemento C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSomma">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSottrazione">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDivisione">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoltiplicazione">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInverso">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inverti A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConiugato">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Coniugato A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNorma">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Norma A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoveToA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sposta in A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoveToB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sposta in B</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtnComplex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Complessi</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtnQuaternion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Quaternioni</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
